### DISCUSSION TITLE
General-dev

### DISCUSSION THREAD

**Discussion by N-R-K** [resurrecting-open-source-projects/scrot#325] [General]
This is just a dummy thread for discussing small/mundane development related things which don't warrant their own issue/discussion.

**Comment by N-R-K** [comment]
There's been a decent amount of bug fixes and improvements since last release. I'm thinking of cutting  a new release this week or so. I plan on merging the following PRs before that, any other bugfix/PRs that are worth merging before release?

ping: @guijan 

- - -

- [x] https://github.com/resurrecting-open-source-projects/scrot/pull/317
- [x] https://github.com/resurrecting-open-source-projects/scrot/pull/326

**Reply by guijan** [reply]
If anything, the amount of changes done to the code means we've gone 2 steps forward 1 step back in many places. Maybe someone will find the new bugs (if any) while they're fresh if we wait around for a while. I'd moderate releases to once or twice a year regardless of how much activity there was in the timeframe. 

But I don't think any of the PRs at the moment are particularly urgent.

**Reply by N-R-K** [reply]
> the amount of changes done to the code means we've gone 2 steps forward 1 step back in many places.

Don't disagree here.

> Maybe someone will find the new bugs (if any) while they're fresh if we wait around for a while.

But I think you might be overestimating how many people actually run software from `master`, in my experience most people stick to their distro's package. (E.g in the most recent bug report that isn't from a scrot developer https://github.com/resurrecting-open-source-projects/scrot/issues/320 it's using `v1.5` - presumably, because that's what his distro has).

Cutting a release will allow distros to pick it up in their unstable branches (e.g debian sid, gentoo's unstable keyword etc).

**Comment by guijan** [comment]
Maybe we should switch to semver when the first deprecation turns into removal?

**Reply by N-R-K** [reply]
Was thinking the same, though it's a bit more difficult to semver a end user application. For example, the addition of `--compress` can also be considered a breaking change (as it alters the behaviour of previous commands relying on `--quality`).

**Comment by N-R-K** [comment]
@eribertomota We should be ready for a `1.10` release. There's a draft changelog committed in the `master` branch.

**Reply by eribertomota** [reply]
I will take care of this now. Sorry, I only saw your private email message now.

**Comment by N-R-K** [comment]
Hey, I'm thinking we're ready to release `v2.0.0`. Is there anything you guys want to fix/add before tagging a new release? @daltomi @guijan

**Reply by daltomi** [reply]
No, I have nothing to add on my part.
